JapaneseFilter/sections/specific.txt

```diff
@@ -67,6 +67,8 @@ corp.rakuten.co.jp#$#@media (max-width: 768px) { .rf-header { margin-bottom: 25p
 corp.rakuten.co.jp#$##pitari_area { display: none !important; }
 basketballking.jp##div[style="text-align: center; margin:30px 0 30px 0;"]
 basketballking.jp##.container > div.free-html
+healthy-vape.shop##.banner
+healthy-vape.shop##p[class]:has(a > img[src*="/banner/bnr_"])
```

Comment:
Please move the rules to AdGuard Annoyances - self-promo.txt under this section `! SECTION: Self-promo - Regular rules`.

Reply:
Done